### ISSUE TITLE
something about eval

### ISSUE THREAD

**Issue by mrsempress** [corenel/pytorch-adda#24]
I don't know in `test.py lines 25: labels = make_variable(labels).squeeze_()`, why the lables need squeeze_(), and in 'pretrain.py' the labels don't need squeeze(). Maybe because the USPS dataset?
